app/mailers/assigned_to_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class AssignedToMailer < ApplicationMailer
+  def assigned_notification(user, project)
+    template_mail(
+      "ec6823ec-0aae-439b-b2f9-c626809b7c61",
```

Comment:
Do we need to amend this template?

Reply:
Is this a question regarding the contents of the email template?

Reply:
Yeah, not sure if it makes reference to Caseworkers specifically! I can ask in the team.

Reply:
Nope, it doesn't!